recipes/argcomplete/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: argcomplete
+  version: "1.8.2"
+
+source:
+  fn: argcomplete-1.8.2.tar.gz
+  url: https://pypi.python.org/packages/7a/60/72d44d1a4ec99c0e8a47764a65850b08bd6e682e0e70ebab2b84632c3c56/argcomplete-1.8.2.tar.gz
```

Comment:
Can you change that URL to `https://pypi.io/packages/source/a/argcomplete/argcomplete-{{ version }}.tar.gz` in the feedsotck? (Merging this now so we don't have to wait for Travis-CI again. The backlog is quite annoying this past week.)